include/logger/err.hpp

```diff
@@ -0,0 +1,85 @@
+#ifndef ERROR_HPP
+#define ERROR_HPP
+
+#include <exception>
+
+/*
+ *@brief Namespace for logger lib .
+ */
+namespace logger {
+
+/*
+ *@brief Namespace for error code .
+ */
+namespace error_handling {
+
+  /*
+   *@brief Error codes.
+   */
+  const int DATA_EMPTY = 101;
+  const int LOG_CREDENTIALS_EMPTY = 102;
+  const int EXTRACTED_LINE_ZERO = 103;
+
+  /*
+   *@brief Gives error message for particular error type.
+   *@param error_code error code
+   *@retval const char*
+   */
+  const char *GetErrorMessage(const int error_code)
+  {
+    switch (error_code) {
+
```

Comment:
no need of line space

Reply:
Actually i use clang format , still i will check why space issues are there.